sos/report/plugins/hbn.py

```diff
@@ -21,4 +21,20 @@ def setup(self):
             '/etc/mellanox',
             '/var/log/doca/hbn',
             '/var/lib/hbn/etc/cumulus',
+            '/var/log/sfc-install.log',
+            '/usr/lib/udev',             # HBN udev rules
+            '/tmp/sf_devices',          # sf devices renamed by HBN udev rules
+            '/tmp/sfr_devices',         # sf representor devices renamed by HBN udev rules
+            '/tmp/sfc-activated',
+            '/tmp/.BR_*',               # temp files created for updating hbn.conf
+            '/tmp/.ENABLE_*',
+            '/tmp/.LINK_*',
+        ])
+
+        self.add_journal(units="sfc")
+        self.add_journal(units="sfc-state-propagation")
+
+        self.add_cmd_output([
+            "mlnx-sf -a show",
```

Comment:
who is installing mlnx-sf? if it's a tool that you need to run then you need to add it to `packages`

Reply:
since you are checking for hbn-repo, we know that it is a system with HBN installed, which implies it is a DPU.
mlnx-sf is part of the base DPU software. But the package is mlnx-tools. I will add.